Aptoide/aptoide-client-v8
# Feature/asv 1529 wallet install offer notification

**What does this PR do?**

This PR aims to implement a new notification to be sent 5 minutes after an app with wallet promotion reaches the claimable state. 

**Database changed?**

   Yes - RealmLocalNotificationSync added

**Where should the reviewer start?**

- [x] NotificationCenter.java
- [x] SystemNotificationShower.java

**How should this be manually tested?**

Change the time to something more reasonable for testing and download an app with promotion+wallet. This should send a notification after both installs are done. Note that claiming the promotion should cancel a scheduled notification that hasn't yet showed;

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1529](https://aptoide.atlassian.net/browse/ASV-1529)

**Code Review Checklist**
